ansible/memcached-operator/build/test-framework/Dockerfile

```diff
@@ -6,12 +6,18 @@ USER 0
 # yum clean all && rm -rf /var/yum/cache/* first
 RUN yum clean all && rm -rf /var/cache/yum/* \
  && yum -y update \ 
- && yum install -y python36-devel python36-pip gcc libffi-devel
+ && yum install -y python36-devel gcc libffi-devel
+
+# ubi7
+RUN if $(cat /etc/redhat-release | grep --quiet 'release 7'); then (yum install -y python36-pip || true); fi
+# ubi8
+RUN if $(cat /etc/redhat-release | grep --quiet 'release 8'); then (yum install -y python3-pip || true); fi
```

Comment:
Didn't we finish the transition to UBI8? If so, no need for the UBI7 stuff to keep sticking around, right?. Ditto for the `operator-sdk` repo.

Reply:
See : https://github.com/operator-framework/operator-sdk-samples/pull/80. 
I will do the same here.

Reply:
Sounds good! Let's wait to see if CI passes on operator-framework/operator-sdk#2104, and then this LGTM